### PR TITLE
Opentok v2.13.0 support for patching archives & broadcasts

### PR DESCRIPTION
**Pull request by woodja**
Adds support for adding/removing streams in an active recording.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://tokbox.com/developer/rest/#http-patch-to-archive-streams

### CODE REVIEW

File: types/opentok/index.d.ts [hunk 1]
```diff
@@ -69,6 +70,13 @@ declare module 'opentok' {
       hasVideo: boolean;
     }
 
+    export interface PatchStream {
+      addStream?: string;
+      removeStream?: string;
+      hasAudio: boolean;
+      hasVideo: boolean;
```
Comment (MysteryBlokHed) [inline types/opentok/index.d.ts:77]: I think these properties are meant to be optional (<https://tokbox.com/developer/rest/#patch-data>)

```suggestion
      hasAudio?: boolean | undefined;
      hasVideo?: boolean | undefined;
```

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Comment by woodja** [comment]
Adjusted implementation to use wrapper methods instead of patch operation directly. https://tokbox.com/developer/sdks/node/reference/OpenTok.html#addBroadcastStream
https://tokbox.com/developer/sdks/node/reference/OpenTok.html#addArchiveStream
...

**Review by MysteryBlokHed — APPROVED** [review]
New changes look good!

**Pull request merged**